playground/terraform/build.gradle.kts

```diff
@@ -247,24 +247,6 @@ tasks {
     }
 }
 
-/* set Docker Registry to params from Inf */
-tasks.register("setDockerRegistry") {
```

Comment:
We use, but we are expecting that it will be passed through the command line. The issue is that this value is used by the Gradle before we can reliably read it from terraform configuration.
For example, [this](https://github.com/apache/beam/blob/f333e0f8cd3a3771c0d8ce703fdf857ca2fa83c7/playground/backend/containers/python/build.gradle#L76) line will be evaluated before any of the tasks are executed. We can be reading terraform in the gradle configuration phase, but it's not reliable as there is no way to define the order in which dependencies are evaluated